.github/workflows/CI.yaml

```
@@ -58,6 +58,8 @@ jobs:
 
       - name: CodeCov
         uses: codecov/codecov-action@v3
+        env:
+          CODECOV_TOKEN = ${{ secrets.CODECOV_TOKEN }}
```

CI is failing currently and I saw elsewhere that they used a `:` instead of a `=`, but not sure if that is the problem...
```suggestion
          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```